opm/material/fluidsystems/blackoilpvt/LiveOilPvt.hpp

```diff
@@ -166,9 +165,9 @@ class LiveOilPvt
         }
 
         vapPar2_ = 0.0;
-        if (deck.hasKeyword("VAPPARS")) {
-            const auto& vapParsKeyword = deck.getKeyword("VAPPARS");
-            vapPar2_ = vapParsKeyword.getRecord(0).getItem("OIL_DENSITY_PROPENSITY").template get<double>(0);
+        const auto& oilvap = eclState.getSchedule().getOilVaporizationProperties(0);
```

Comment:
again, I'd argue the New Way is harder to understand: to me it is unclear that `getOilVaporizationProperties()` represents the `VAPPARS` keyword and I would not expect it to be part of the schedule...